### ISSUE TITLE
cp: cannot create regular file: mt4-tester.ini: No such file or directory

### ISSUE THREAD

**Issue by kenorb** [EA31337/FX-MT-VM#98]
```
vagrant@vagrant:~$ /vagrant/scripts/run_backtest.sh -v -e MACD -y 2014 -m 1 -C -R -M4.0.0.971
Copying ini files...
'/vagrant/conf/mt4-tester.ini' -> '/home/vagrant/.wine/drive_c/users/vagrant/Application Data/MetaQuotes/WebInstall/mt4clw/config/mt4-tester.ini'
cp: cannot create regular file '/home/vagrant/.wine/drive_c/users/vagrant/Application Data/MetaQuotes/WebInstall/mt4clw/config/mt4-tester.ini': No such file or directory
wineserver is /usr/bin/wineserver
Exiting /vagrant/scripts/run_backtest.sh with 1
```

Est. 1-3h

**Comment by kenorb** [comment]
It seems it works.

**Comment by kenorb** [comment]
Most likely fixed as part of PR-#99.
